app/src/main/res/layout/activity_timeline.xml

```diff
@@ -10,23 +10,32 @@
     tools:context=".TimelineActivity">
 
     <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
         android:id="@+id/swipeContainer"
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/rvTweets"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/rvTweets"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
 
     </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
     <Button
         android:id="@+id/logoutButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignParentEnd="true"
-        android:layout_marginEnd="25dp"
+        android:layout_alignParentStart="true"
+        android:layout_marginStart="310dp"
+        android:layout_marginEnd="-7dp"
+        android:layout_marginBottom="459dp"
```

Comment:
nit: These should ideally go into xml resources.
https://github.com/jjojan/Twitter/blob/main/app/src/main/res/values/dimens.xml